cypress/cypress/views/clusterset/clusterset.js

```diff
@@ -9,6 +9,7 @@ import { acmHeaderSelectors } from '../header'
 import { commonElementSelectors, commonPageMethods } from '../common/commonSelectors'
 
 import { clustersPages } from '../clusters/managedCluster'
+import { getClusterSet } from '../../apis/clusterSet'
```

Comment:
I see another import of the "api/clusterSet" here.
And it looks like it is used in lines 53 and 71.

So, in case we need the "api/clusterSet" import, please add it.

Reply:
You are right, I will add it.